Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream> 
+#include "Person.h"
+
+
+
+
+std::string(n);
```

Comment:
If you want to make a string variable in C++, syntax is like `std::string n;` similar to when you make a `int` or `char` variable just that for string it just need the `std::` because it is part of the library. I think you also need to add `#include <string>` to your includes.